spec/lib/miq_expression_spec.rb

```diff
@@ -2480,8 +2480,8 @@
     end
   end
 
-  describe ".get_col_type" do
-    subject { described_class.get_col_type(@field) }
+  describe ".parse_field_or_tag" do
```

Comment:
I think we already have these tests for `parse_field_or_tag`, right?

Reply:
I checked it and we don't. It was probably because we have test for `MiqExpression::Target` subclasses (`Field`, ...)